components/dashboard/src/components/ProgressBar.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC } from "react";
+
+type Props = {
+    percent: number;
+};
+export const ProgressBar: FC<Props> = ({ percent }) => {
+    return (
+        <div className="w-full rounded-full h-2.5 bg-gray-300 dark:bg-gray-300">
+            <div
+                className="transition-width ease-linear duration-1000 bg-blue-500 h-2.5 rounded-full"
+                style={{ width: `${percent}%` }}
+            />
+        </div>
+    );
+};
```

Comment:
**thought**: Not sure how much the progress bar helps. Is this a determinate or indeterminate progress bar? Do we know how much time it takes to finish the process?

Reply:
It's determinate, when we start exporting, we know how many pages of data to load, so the progress indicator is based off of how many pages we've loaded so far. We could probably estimate the duration fairly well (with optimistic conditions), but I'm not sure it's worth trying to do that quite yet.

In playing with a larger dataset, like grabbing gitpod's year to date, it takes awhile, which made the progress bar feel pretty helpful/reassuring. I'd suggest we keep it while we have this behind a flag, and try it out with larger datasets that prod has, and we can easily remove it if we still don't think it adds value. How does that sound?

Reply:
Sounds perfect. 😃